llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -18035,6 +18053,7 @@ RISCVTargetLowering::EmitInstrWithCustomInserter(MachineInstr &MI,
            "ReadCounterWide is only to be used on riscv32");
     return emitReadCounterWidePseudo(MI, BB);
   case RISCV::Select_GPR_Using_CC_GPR:
+  case RISCV::Select_IMM_Using_CC_GPR:
```

Comment:
This is misnamed. It should be `Select_GPR_Using_CC_Imm`. The first type is the of the register the instruction produces.